example/src/main/java/com/xwray/groupie/example/item/CarouselItem.kt

```diff
@@ -18,11 +19,17 @@ class CarouselItem(private val carouselDecoration: RecyclerView.ItemDecoration,
         return R.layout.item_carousel
     }
 
+    override fun createViewHolder(itemView: View): GroupieViewHolder {
+        return super.createViewHolder(itemView).apply {
+            recyclerView.apply {
+                layoutManager = LinearLayoutManager(context, LinearLayoutManager.HORIZONTAL, false)
+                adapter = carouselAdapter
```

Comment:
This will cause an issue if you have more than one `CarouselItem` in the adapter and you rebind the item. It means both will share the adapter of the first one instead of the 2nd one having a new adapter set because `createViewHolder` will only have been called once.